docs/architecture-decisions/adr013-use-fetch.md

```diff
@@ -12,13 +12,13 @@ support burden of keeping said package up to date.
 
 ## Decision
 
-Backend (node) packages should use the `node-fetch` package for HTTP data
-fetching. Example:
+Backend (node) packages should use the native `fetch` for HTTP data fetching,
+and importing `undici` where necessary. Example:
```

Comment:
Why would one need to do this? thinking it's good to have an example or just not mention the option

Reply:
Maybe it's not worth mentioning in here honestly. It's for stuff like `setGlobalDispatcher` and `ProxyAgent` usage etc ... and maybe the only use case really is in the corporate proxy document. So ... remove?